x/auth/ante.go

```diff
@@ -94,8 +95,15 @@ func NewAnteHandler(am AccountMapper, fck FeeCollectionKeeper) sdk.AnteHandler {
 				return newCtx, res, true
 			}
 
+			if requiredFees := adjustFeesByGas(ctx.MinimumFees(), fee.Gas); ctx.IsCheckTx() && !simulate &&
```

Comment:
This is not easy to read. May I suggest:

```golang
requiredFees := adjustFeesByGas(ctx.MinimumFees(), fee.Gas)
if !simulate && ctx.IsCheckTx() && !ctx.MinimumFees().IsZero() && !fee.Amount.IsGTE(requiredFees) {
    // ...
}
```

Reply:
ACKd, committed and pushed.